bazel/emscripten_toolchain/crosstool.bzl

```diff
@@ -1070,18 +1043,10 @@ def _impl(ctx):
     features.append(crosstool_default_flags_feature)
 
     cxx_builtin_include_directories = [
-        emscripten_version + "/system/lib/libc/musl/arch/emscripten",
-        emscripten_version + "/system/lib/libc/musl/arch/js",
-        emscripten_version + "/system/local/include",
-        emscripten_version + "/system/include/compat",
-        emscripten_version + "/system/include",
-        emscripten_version + "/system/include/libcxx",
-        emscripten_version + "/system/lib/compiler-rt/include",
-        emscripten_version + "/system/lib/libcxxabi/include",
-        emscripten_version + "/system/include/libc",
-        emscripten_version + "/system/include/gfx",
-        emscripten_version + "/system/include/SDL",
-        emscripten_version + "/lib/clang/12.0.0/include",
+        "external/emscripten/lib/clang/12.0.0/include",
+        "external/emscripten/emscripten/cache/sysroot/include",
+        "external/emscripten/emscripten/cache/sysroot/include/c++/v1",
+        "external/emscripten/emscripten/cache/sysroot/include/compat",
```

Comment:
what about .../sysroot/local/include (to match above) ? 

It would be nice if they were consistently ordered, but that might be a bit onerous.

Reply:
Actually I think local should not be needed in either case.  We use local to allow folks to install their own headers.. but in this case there is no way to write the sysroot anyway.